### ISSUE TITLE
keepassxc-browser fill the password in the username field

### ISSUE THREAD

**Issue by sebastienserre** [keepassxreboot/keepassxc-browser#1229]
Hello,

When I use keepassxc-browser the password is filled in the username fields and the password one stay empty.
![keepassxc](https://user-images.githubusercontent.com/3902621/108472984-b9514b80-728d-11eb-8802-bcad70ac8102.png)

## Expected Behavior
Be able to fill the right info in the right field

## Debug info
KeePassXC - 2.6.4 (PPA)
KeePassXC-Browser - 1.7.6
Operating system: Linux x86_64 // Ubuntu 20.04
Browser: Mozilla Firefox 85.0

**Comment by varjolintu** [comment]
What is the site where this happens?

**Comment by sebastienserre** [comment]
I tested on https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr and https://thiverval-grignon.com/wp-admin/
I've tested on the WordPress login which is a more simple page with only 2 fields but still same. The password is filled in the username.

Hummm, writing this blogpost, I realized I've imported from LastPass CSV and I think there were a matter on the import and csv has been mis simported.
Sorry for this, I will better check my import.

Thanks for your job
